src/Common/Macros.h

```diff
@@ -57,8 +57,6 @@ class Macros
 
     String expand(const String & s) const;
 
-    String expand(const String & s, const StorageID & table_id, bool allow_uuid) const;
```

Comment:
This has been handled by https://github.com/ClickHouse/ClickHouse/blob/105308ee545d0c80d3a3662ba7bb26b3cf79a5ff/src/Storages/MergeTree/registerStorageMergeTree.cpp#L318